databricks/koalas/missing/frame.py

```diff
@@ -31,15 +31,15 @@ class _MissingPandasLikeDataFrame(object):
 
     # Properties
     axes = unsupported_property('axes')
-    ftypes = unsupported_property('ftypes')
     iat = unsupported_property('iat')
-    is_copy = unsupported_property('is_copy')
-    ix = unsupported_property('ix')
     ndim = unsupported_property('ndim')
     style = unsupported_property('style')
 
     # Deprecated properties
     blocks = unsupported_property('blocks', deprecated=True)
+    ftypes = unsupported_property('ftypes', deprecated=True)
+    is_copy = unsupported_property('is_copy', deprecated=True)
```

Comment:
`is_copy` is deprecated in the codes (see https://github.com/pandas-dev/pandas/blob/269d36814e471880627109a5e5ea63d4a5fdc5a6/pandas/core/generic.py#L210-L212)